arraycontext/impl/pytato/compile.py

```diff
@@ -636,10 +636,14 @@ def __call__(self, arg_id_to_arg) -> ArrayContainer:
         input_kwargs_for_loopy = _args_to_device_buffers(
                 self.actx, self.input_id_to_name_in_program, arg_id_to_arg, fn_name)
 
-        _evt, out_dict = self.pytato_program(queue=self.actx.queue,
+        evt, out_dict = self.pytato_program(queue=self.actx.queue,
                                             allocator=self.actx.allocator,
                                             **input_kwargs_for_loopy)
 
+        if self.actx.profile_kernels:
+            from arraycontext.impl.pytato import ProfileEvent
+            self.actx.profile_events.append(ProfileEvent(evt, self.pytato_program))
```

Comment:
Add a `self.actx.add_profiling_event` or something like that? Since there's already a `get_profiling...`.